aws/cloudformation/iam.yml.erb

```
@@ -835,6 +835,90 @@ roles.each do |name, config| %>
     Type: AWS::IAM::ServiceLinkedRole
     Properties: 
       AWSServiceName: logger.cloudfront.amazonaws.com
+
+  # ------------------------------------------------------------
+  # Create one AWS IAM User for each type of marketing sites environment that GitHub Actions will create, with
+  # permissions needed to create/update/delete a marketing sites CloudFormation Stack.
+  # ------------------------------------------------------------
+  <%
+    %w(development test production).each do |environment_type|
+  %>
+  GithubUser<%=environment_type%>:
+    Type: AWS::IAM::User
+    Properties:
+      UserName: "github-actions-marketing-sites-<%=environment_type%>"
+      PermissionsBoundary: !Ref DevPermissions
+  GithubPolicy<%=environment_type%>:
```

nit: name policies based on what actions they allow, not based on who/what uses them.

```suggestion
  DeployMarketingSites<%=environment_type%>:
```

(if you edit this, also edit PolicyName below to match)

---

Done!